src/app/components/filter/filter.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { MatSort } from '@angular/material/sort';
+import { SortWord } from '../../shared/interfaces';
+
+@Component({
+  selector: 'app-filter',
+  templateUrl: './filter.component.html',
+  styleUrls: ['./filter.component.scss']
+})
+export class FilterComponent {
+  @Input() public show: Boolean;
+  @ViewChild(MatSort) public sort: MatSort;
+  @Output() public sortInfo: EventEmitter<SortWord> = new EventEmitter<SortWord>();
+  public inputValue: string;
```

Comment:
it is better to put it the next way:
1. @Input
empty line
2. @Output
empty line
3. @ViewChild
empty line
...  
other fields